sessionctx/binloginfo/binloginfo_test.go

```diff
@@ -614,6 +614,14 @@ func (s *testBinlogSuite) TestAddSpecialComment(c *C) {
 			"create table t1 (id int, a varchar(255), primary key (a, b) clustered);",
 			"create table t1 (id int, a varchar(255), primary key (a, b) /*T![clustered_index] clustered */ );",
 		},
+		{
+			"create table t1(id int, v int, primary key(a) clustered);",
+			"create table t1(id int, v int, primary key(a) /*T![clustered_index] clustered */ );",
+		},
+		{
+			"create table t1(id int primary key clustered, v int);", // CDC upstream => downstream never produce this
```

Comment:
Should we consider `ALTER TABLE t ADD PRIMARY KEY CLUSTERED`?

Reply:
good catch.. and it seem cdc also will receive `create table t1(id int primary key clustered` if upstream do that..

and it seems we didn't support add pk with clustered..now

```
MySQL [test]> alter table tx add primary key(a) clustered;
Query OK, 0 rows affected (2.526 sec)

MySQL [test]> show create table tx;
+-------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| Table | Create Table                                                                                                                                                                                     |
+-------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| tx    | CREATE TABLE `tx` (
  `a` bigint(20) NOT NULL,
  `b` int(11) DEFAULT NULL,
  PRIMARY KEY (`a`) /*T![clustered_index] NONCLUSTERED */
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_bin |
+-------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
1 row in set (0.001 sec)

```